draft-ietf-quic-transport.md

```diff
@@ -3795,119 +3819,84 @@ Handshake encryption level is discarded - and no longer retransmitted - when
 Handshake protection keys are discarded.
 
 
-## Retry Packet {#packet-retry}
-
-A Retry packet uses a long packet header with a type value of 0x3. It carries
-an address validation token created by the server. It is used by a server that
-wishes to perform a stateless retry (see {{validate-handshake}}).
+## Short Header Packet {#short-header}
```

Comment:
I'm assuming that the diff here looks terrible because it's moved text, not changed text.

Reply:
Yep.